libbeat/tests/system/test_ca_pinning.py

```diff
@@ -39,7 +39,7 @@ def test_sending_events_with_a_good_sha256(self):
                 # Use certificate verification only to avoid validating localhost as the hostname.
                 "ssl_verification_mode": "certificate",
```

Comment:
@belimawr should we fix it in the same way as in https://github.com/elastic/beats/pull/34501/commits/76256ffb53d7fb06c2837c3660e024de67ff3e1a ?